config/config.yml

```diff
@@ -479,3 +479,7 @@ PERMITTED_HOSTS: [
 ]
 
 USER_RENAME_LIMIT_DAYS: 7
+
+# Number of seconds to retain the original creator for works that have been
```

Comment:
Can you change the units in the comment to hours? It looks like you added the units to the variable name, but the comment still says seconds.

Reply:
Thanks for the catch! Can't believe I missed it earlier :facepalm: